pinot-common/src/main/java/org/apache/pinot/common/assignment/InstancePartitionsUtils.java

```diff
@@ -63,14 +64,20 @@ public static InstancePartitions fetchOrComputeInstancePartitions(HelixManager h
 
     // Fetch the instance partitions from property store if it exists
     ZkHelixPropertyStore<ZNRecord> propertyStore = helixManager.getHelixPropertyStore();
-    InstancePartitions instancePartitions = fetchInstancePartitions(propertyStore,
-        getInstancePartitionsName(tableNameWithType, instancePartitionsType.toString()));
+    InstancePartitions instancePartitions;
```

Comment:
self-review: I shouldn't return instance partitions for group, instead first look-up table's instance partitions and if they aren't available only the return group instance partitions (after changing their name).